src/controllers/beneficial.owner.other.controller.ts

```diff
@@ -1,141 +1,29 @@
 import { NextFunction, Request, Response } from "express";
-import { logger } from "../utils/logger";
-import { Session } from "@companieshouse/node-session-handler";
-import { saveAndContinue } from "../utils/save.and.continue";
-import { ApplicationDataType } from "../model";
-import { getFromApplicationData, mapDataObjectToFields, mapFieldsToDataObject, prepareData, removeFromApplicationData, setApplicationData } from "../utils/application.data";
-import { BeneficialOwnerOtherKey, BeneficialOwnerOtherKeys } from "../model/beneficial.owner.other.model";
-import {
-  AddressKeys,
-  BeneficialOwnerNoc,
-  HasSamePrincipalAddressKey,
-  ID,
-  InputDateKeys,
-  IsOnRegisterInCountryFormedInKey,
-  IsOnSanctionsListKey,
-  NonLegalFirmNoc, PublicRegisterNameKey, RegistrationNumberKey,
-  TrusteesNoc
-} from "../model/data.types.model";
-import { PrincipalAddressKey, PrincipalAddressKeys, ServiceAddressKey, ServiceAddressKeys } from "../model/address.model";
-import { StartDateKey, StartDateKeys } from "../model/date.model";
-import { v4 as uuidv4 } from 'uuid';
 import { BENEFICIAL_OWNER_OTHER_PAGE, BENEFICIAL_OWNER_TYPE_URL } from "../config";
+import {
+  getBeneficialOwnerOther,
+  getBeneficialOwnerOtherById,
+  postBeneficialOwnerOther,
+  removeBeneficialOwnerOther,
+  updateBeneficialOwnerOther
+} from "../utils/beneficial.owner.other";
 
 export const get = (req: Request, res: Response) => {
-  logger.debugRequest(req, `GET ${BENEFICIAL_OWNER_OTHER_PAGE}`);
```

Comment:
@mouhajer-ch , we had to extract the common code into a separate file as we were displaying the update journey screens in a separate template file, if you need this to be reverted ,can you disable the duplication checks on sonar(not sure if this will exceed the 3% limit on sonar)
@AndreKramer7654 / @bantao-ch , could I have your inputs here please

Reply:
Improvement will be required